### ISSUE TITLE
Checkpointing strategy

### ISSUE THREAD

**Issue by Arkkienkeli** [cytomining/DeepProfiler#231]
Add a config parameter to change checkpointing step (now checkpoints are saved for every epoch) or save the best validation accuracy checkpoint or both. 

- Update code
- Update wiki

**Comment by jccaicedo** [comment]
Sounds good!
I think it would be great to support the same alternatives that Keras has. In which section of the config file are you planning on adding this parameter? And what would be the name?

**Comment by Arkkienkeli** [comment]
I think I will put it into `train` -> `model` section.
Probably, it should be named as `frequency` or `checkpoint_freq` as we already have similar option in the `validation` section.

**Comment by jccaicedo** [comment]
Perfect. I think `train` -> `model` makes sense. And the name could be `checkpoint_policy` 👍

**Comment by Arkkienkeli** [comment]
b10b4427b3c883ccac46a6196b469dac6dc1077e
